### PR TITLE
Add default handlers for showInputBox and showQuickPick

### PR DESCRIPTION
**Pull request by timfel**
These are a VSCode specific, but since there are already minor handlers for window messages that aren't strictly part of the LSP, and some servers (like Netbeans Java Language Server) use these to request additional info for refactorings, I thought it might be useful in general

### CODE REVIEW

File: lsp-mode.el [hunk 1]
```diff
@@ -1914,6 +1914,29 @@ regex in IGNORED-FILES."
            (indent 1))
   `(when-let ((lsp--cur-workspace ,workspace)) ,@body))
 
+(lsp-defun lsp--window-show-quick-pick (_workspace (&ShowQuickPickParams :place-holder :can-pick-many :items))
+  (if-let* ((selectfunc (if can-pick-many #'completing-read-multiple #'completing-read))
+            (itemLabels (cl-map 'list
```
Comment (yyoncho) [inline lsp-mode.el:1919]: nit: use seq-map

File: lsp-mode.el [hunk 2]
```diff
@@ -1914,6 +1914,29 @@ regex in IGNORED-FILES."
...
+                                items))
+            (result (funcall-interactively
+                     selectfunc
+                     (format "%s%s " place-holder (if can-pick-many " (* for all)" "")) itemLabels))
+            (choices (if (listp result)
+                         (if (equal result '("*"))
+                             itemLabels
+                           result)
+                       (list result))))
+      (vconcat (seq-filter #'identity (cl-map 'list
```
Comment (yyoncho) [inline lsp-mode.el:1930]: ditto

**Review by yyoncho — COMMENTED** [review]
Looks good. Some minor nits. Also CI failed with `picked` not used

File: lsp-mode.el (hunk 2 above)
Comment (timfel) [reply lsp-mode.el]: done

File: lsp-mode.el (hunk 1 above)
Comment (timfel) [reply lsp-mode.el]: done

**Comment by yyoncho** [comment]
CI is still red:

```
In toplevel form:
lsp-mode.el:89[14](https://github.com/emacs-lsp/lsp-mode/runs/5124172248?check_suite_focus=true#step:11:14):1:Error: the function ‘lsp--window-input-box’ is not known to be defined.
make: *** [Makefile:43: unix-compile] Error 1
```

**Comment by timfel** [comment]
> CI is still red:

Ooops, stupid typo

**Review by yyoncho — APPROVED** [review]

**Pull request merged**